maps-compose/src/main/java/com/google/maps/android/compose/GoogleMap.kt

```diff
@@ -74,6 +74,7 @@ import kotlinx.coroutines.awaitCancellation
 @Composable
 public fun GoogleMap(
     modifier: Modifier = Modifier,
+    currentLocationModifier : Modifier = Modifier,
```

Comment:
Having two Modifiers is hard to understand to a user and makes the already rich GoogleMap() API surface even heavier.

This is needed only because of the changes introduced in #347. I pointed out several problems in https://github.com/googlemaps/android-maps-compose/issues/340#issuecomment-1753175867 and https://github.com/googlemaps/android-maps-compose/issues/340#issuecomment-1753252034.

My suggestion is to move the changes from #347 into an official example instead of including them in the API to address these problems and to reduce API bloat.

I understand #347 is your baby, but please take a step back and reconsider the options. Adding custom buttons in the fashion of #347 makes for a poor API experience.

Reply:
Thank you for explaining the rationale, @bubenheimer. We are merging this now as a breaking change.